core/src/main/java/com/oath/oak/OakMapBuilder.java

```diff
@@ -43,6 +44,7 @@ public OakMapBuilder() {
         this.chunkMaxItems = Chunk.MAX_ITEMS_DEFAULT;
         this.memoryCapacity = MAX_MEM_CAPACITY;
         this.memoryAllocator = null;
+        this.operator = new ValueUtilsImpl();
```

Comment:
I do not think builder should create operator... If it is one per Oak it should be in InternalOakMap or a singleton

Reply:
I think that just as the memory manager is created in the Builder, so should the valueOperator.
I think, however, that there should be a method to set the valueOperator, so making the access to the off-heap lock-free would be seamless.